modules/runtime/nuxeo-runtime/src/main/java/org/nuxeo/runtime/services/config/ConfigurationServiceImpl.java

```diff
@@ -191,9 +148,8 @@ protected static boolean startsWithNamespace(String string, String namespace) {
      */
     @Override
     public Optional<String> getString(String key) {
-        return Optional.ofNullable(getDescriptors().get(key))
-                       .map(ConfigurationPropertyDescriptor::getValue)
-                       .filter(StringUtils::isNotBlank);
+        Optional<ConfigurationPropertyDescriptor> desc = getRegistryContribution(CONFIGURATION_EP, key);
+        return Optional.ofNullable(desc.map((value) -> value.getValue()).filter(StringUtils::isNotBlank).orElse(null));
```

Comment:
This seems weird. Isn't this simpler?

    desc.map(value -> value.getValue()).filter(StringUtils::isNotBlank);